packages/utilities/fast-web-utilities/src/strings.ts

```diff
@@ -41,7 +41,7 @@ export function startsWith(
         return false;
     }
 
-    return stringToSearch.substr(position, searchFor.length) === searchFor;
+    return stringToSearch.substring(position, searchFor.length) === searchFor;
```

Comment:
I don't think a change like this is trivial - technically this change could constitute a breaking change. While `substr() ` and `substring()` are similar, they are different and could result in differences in what's being returned: https://stackoverflow.com/a/3745518/4782111

Reply:
Thanks for pointing out me @chrisdholt. I thought I've replaced a deprecated method with a working one. Though now I have a look at what I have done. Will try to look forward from next time onwards before changing anything.